browser/brave_browser_process_impl.cc

```diff
@@ -37,7 +37,9 @@
 #include "brave/services/network/public/cpp/system_request_handler.h"
 #include "chrome/browser/component_updater/component_updater_utils.h"
 #include "chrome/browser/net/system_network_context_manager.h"
+#if !defined(OS_ANDROID)
```

Comment:
this should be placed after the main list like the buildflags below

Reply:
actually in here https://github.com/brave/brave-core/pull/6554/commits/7b4b34ac0186963df472ae74b0e6b8211c44ec73#diff-455efc84fa098a0757b0f2232a5b1f473b9d2c6b5ed13d5665fc5dc9bb9aa46cR91

Reply:
Fixed in 1105be6d38f16175dba23eef73a248f35c2819e3